plugins/processors/awsappsignals/common/types.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package common
+
+const (
+	AttributeRemoteService       = "aws.remote.service"
+	AttributeHostedInEnvironment = "aws.hostedin.environment"
+)
+
+const (
+	MetricAttributeRemoteNamespace = "K8s.RemoteNamespace"
+	MetricAttributeLocalService    = "Service"
+	MetricAttributeLocalOperation  = "Operation"
+	MetricAttributeRemoteService   = "RemoteService"
+	MetricAttributeRemoteOperation = "RemoteOperation"
+	MetricAttributeRemoteTarget    = "RemoteTarget"
+)
+const (
+	HostedInAttributeClusterName  = "HostedIn.EKS.Cluster"
+	HostedInAttributeK8SNamespace = "HostedIn.K8s.Namespace"
+	HostedInAttributeEnvironment  = "HostedIn.Environment"
+)
```

Comment:
Do we not also want `HostedIn.K8s.Cluster` now?

Reply:
Followed up offline, team will take a followup action to add any attributes needed for new platform support